src/commands.cpp

```diff
@@ -324,6 +324,15 @@ CommandPutFile::CommandPutFile(MegaClient* client, TransferSlot* ctslot, int ms)
     arg("v", 2);
     arg("s", tslot->fa->size);
     arg("ms", ms);
+
+    for (auto &file : tslot->transfer->files)
+    {
+        if (!ISUNDEF(file->h))
+        {
+            arg("t", (byte*)&file->h, MegaClient::NODEHANDLE);
+            break;
+        }
+    }
```

Comment:
If I upload the same large file to multiple folders, it's just one transfer with multiple entries in the files list.   However some targets could be in the user's account, some could be in inshares.   Also multiple targets in the user's own account could take it overquota when only one would not take it overquota.   Shouldn't we take all the entries in the files list into account?   thanks